tid_project/src/api/config.js

```diff
@@ -0,0 +1,8 @@
+export const API_CONFIG = {
+  baseURL: 'https://parseapi.back4app.com',
+  headers: {
+    'X-Parse-Application-Id': 'BLJvJPeABAqvYm1193o5WAfaCEpfzvjAuDTLqe2P',
+    'X-Parse-JavaScript-Key': 'Tf7tdCcH6j3YCJkzRJp05VcLIddIzGtbAs6rGruN',
```

Comment:
For better security, you may want to define env constants for X-Parse-Application-Id and  X-Parse-JavaScript-Key